keps/sig-storage/20181111-extend-datasource-field.md

```diff
@@ -114,6 +114,8 @@ Currently the CSI provisioner already accepts the DataSource field in new provis
 
 To emphasize above, this feature will ONLY be available for CSI.  This feature wil NOT be added to In-tree plugins or Flex drivers, this is strictly a CSI only feature.
 
+It's important to note that we intentionally require that a source PVC be in the same StorageClass as the PVC being created.  This is currently required because the StorageClass determintes characteristics for a volume like `fsType`.  Performing a clone from an xfs volume to an ext4 volume for example would not be acceptable, given that a storageClass can have unique information, cloning across storage classes is not something we're able to try and determine safely at this time.
```

Comment:
Maybe this would be more visible under non-goals?

Reply:
Added a note in "non-goals" with a link, seems like a good fit, let me know what you think.